src/store/editor-slice.ts

```diff
@@ -165,7 +165,7 @@ function createEditorReducer(
 					state.editors[payload.editorId] = {
 						...empty,
 						text: textWithSignatureRepliesForwards,
-						subject: `Fwd: ${payload.original.subject}`,
+						subject: `FWD: ${payload.original.subject.replace(/(^(fwd:\s)+)/i, '')}`,
```

Comment:
we generally prefer defining the constant for Regex

Reply:
moved! ty!